CONTRIBUTING.md

```diff
@@ -153,6 +153,13 @@ required to ensure that exported user records contain the password hashes of the
 3. Click 'ADD ANOTHER ROLE' and choose 'Firebase Authentication Admin'.
 4. Click 'SAVE'.
 
+For some of the authentication integration tests, it is required to manually enable Email/Password 
```

Comment:
For some of the Firebase Auth integration tests, it is required to enable the Email/Password sign-in method: